packages/gestalt/src/Contents.js

```diff
@@ -10,11 +10,12 @@ import { useColorScheme } from './contexts/ColorScheme.js';
 
 /* Needed until this Flow issue is fixed: https://github.com/facebook/flow/issues/380 */
 /* eslint quote-props: 0 */
+// prettier-ignore
```

Comment:
Most controversial change - due to https://github.com/facebook/flow/issues/380 not being fixed